### ISSUE TITLE
Keras implementation is not RL

### ISSUE THREAD

**Issue by itschenxi** [junxiaosong/AlphaZero_Gomoku#132]
It seems the Keras implementation does not use policy gradient algorithm (reinforcement learning), instead it uses supervised learning. The Pytorch implementation uses reinforcement learning. Why?
